data/systemd/snap-repair.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Timer to automatically fetch and run repair assertions
```

Comment:
@mvo5 is there a way to not have this, or disable this on classic? or will we get a always failing timer there? or is that intentional so we can change our mind later if we want to run this on classic too?